frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -1008,9 +1006,12 @@ class TracingApi {
    */
   getTreeName(treeId?: number) {
     const tracing = assertSkeleton(Store.getState().annotation);
-    return getTree(tracing, treeId)
-      .map((activeTree) => activeTree.name)
-      .get();
+    const treeName = getTree(tracing, treeId)?.name;
```

Comment:
`maybe.get()` did throw an error if the request value was `maybe.Nohing`:

https://folktalegithubio.readthedocs.io/en/latest/api/data/maybe/Maybe.html#Maybe.prototype.get